spark_client/services.py

```diff
@@ -275,6 +275,11 @@ def select_by_master(self, master: str):
         }
 
         self.logger.debug(f"Clusters API: {dict(api_servers_clusters)}")
+        for _context in self.kube_config["contexts"]:
+            print(_context["name"])
```

Comment:
Why do we still have prints here?

Reply:
oops, this was when I was writing down the tests to check what to assert for. sorry. let me fix these.